frontend/public/components/factory/list-page.jsx

```diff
@@ -240,6 +240,7 @@ export const FireMan_ = connect(null, { filterList })(
               <Dropdown
                 buttonClassName="pf-m-primary"
                 id="item-create"
+                menuClassName="pf-m-align-right-on-md"
```

Comment:
This caused https://bugzilla.redhat.com/show_bug.cgi?id=1833616.  I think the fix is: 
`menuClassName={classNames('pf-m-align-right-on-md' : !title)}`
but haven't tested as I'm lacking a cluster.  @sg00dwin